stable/insights-agent/README.md

```diff
@@ -86,4 +86,4 @@ Parameter | Description | Default
 `goldilocks.controller.resources` | CPU/memory requests and limits for the Goldilcoks controller |
 `goldilocks.dashboard.enabled` | Installs the Goldilocks Dashboard | false
 `resourcemetrics.installPrometheus` | Install a new Prometheus instance for the resourcemetrics report | false
-`resourcemetrics.address` | The address of an existing Prometheus instance to query | `"http://prometheus-server"`
+`resourcemetrics.address` | The address of an existing Prometheus instance to query in the form `http://<service-name>.<namespace>` | `"http://prometheus-server"`
```

Comment:
probably also include port `http://<service-name>.<namespace>:<service-port>` -> `http://prometheus-server:9090` if

Reply:
ooh and the `http`  in `http://<service-name>.<namespace>` probably change it to `<scheme>` or `<protocol>` may be?

Reply:
Good call